node/src/node_configurator/unprivileged_parse_args_configuration.rs

```diff
@@ -853,6 +853,7 @@ mod tests {
             Ok(NeighborhoodConfig {
                 mode: NeighborhoodMode::Standard(node_addr, _, _),
                 min_hops: Hops::ThreeHops,
+                ..
```

Comment:
Ja bych tady ty tecky odstranil, protoze to rika "at pridame, kolik chceme dalsich fieldu, tak vsechny ignoruj", coz neni zrovna bezpecna vec. Respektive, neni duvod to mit.